scene/gui/color_picker.h

```diff
@@ -93,14 +120,19 @@ class ColorPicker : public BoxContainer {
 	HSeparator *preset_separator = memnew(HSeparator);
 	Button *btn_add_preset = memnew(Button);
 	Button *btn_pick = memnew(Button);
-	CheckButton *btn_hsv = memnew(CheckButton);
-	CheckButton *btn_raw = memnew(CheckButton);
-	HSlider *scroll[4];
-	SpinBox *values[4];
-	Label *labels[4];
+
+	OptionButton *mode_option_button = memnew(OptionButton);
```

Comment:
I guess this works, but we usually create nodes inside constructors. It should be moved and initialized here with nullptr.

Reply:
But other nodes are also initialized here?